tests/test_pipelines/test_variant_calling.py

```diff
@@ -174,11 +184,10 @@ def test_run(self):
                                  '-T RealignerTargetCreator '
                                  '--read_filter BadCigar '
                                  '--read_filter NotPrimaryAlignment '
-                                 '-o tests/assets/jobs/test_dataset/gatk_var_calling/test_user_sample_id.intervals '
-                                 '-l INFO '
+                                 '-o tests/assets/jobs/test_dataset/gatk_var_calling/test_user_sample_id.intervals'
+                                 ' -l INFO '
```

Comment:
Not that this is a real issue but I do not get any failure regardless of where the space is located here. Can you revert back?

Reply:
@tcezard I have reverted as instructed and the assertions started failing again. I have now only removed the trailing whitespace from the string defined on lines 190 and 215, as these strings are the last strings in the set of strings that together make up the command.